shortbreaks/pr-qa-template.md

```diff
@@ -1,12 +1,10 @@
-#### What are the links to original PRs?
+#### What is the link(s)to the original PR(s)?
+
 
 ---
-#### Quality checklist (for PR author to complete BEFORE peer review):
-- [ ] I've checked this work meets the requirements of the Jira.
-- [ ] I am ready for this to be merged to production for further testing and sign-off.
 
-#### @nameOfReviewer 
-- [ ] I agree with my work being merged to Production.
-- [ ] Clear of conflicts.
-- [ ] There have been no changes within this code that the QA has not been made aware of.
-- [ ] +1 from me!
\ No newline at end of file
+By approving this PR I agree that;
+- My work can be merged to Production.
+- Is clear of conflicts.
+- The QA has been made aware of all the changes in the code.
```

Comment:
I think this was clearer the other way round. Made the dev check that no unknown changes had crept in as part of a merge.